ClassRegistration/ClassRegistration.App/Interfaces/IEnrollmentRepo.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ClassRegistration.App.Interfaces
+{
+    /// <summary>
+    /// This interface includes common functionality that is needed by the Enrollment repository
+    /// </summary>
+    public interface IEnrollmentRepo

```

Comment:
Seems like a small thing, but we want to spell out "Repository" fully in class names

Reply:
Resolved.